pom.xml

```diff
@@ -114,6 +114,11 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>tuleap-api</artifactId>
+      <version>1.0</version>
```

Comment:
What do you think about to enforce the use of the latest version of the tuleap-api-plugin here ?
=> The syntax might be [1.0,)   [0]
[0] https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html